chairing.md

```diff
@@ -0,0 +1,48 @@
+# TAG Chairing Guide
+
+See https://www.w3.org/guide/chair/role.html for W3C-wide advice on chairing.
+
+## Choosing a chair
+
+The Process describes [how we pick a
+chair](https://www.w3.org/policies/process/#tag-participation). This may change
+as a result of [w3c/process#891](https://github.com/w3c/process/issues/891) or
+other discussions.
+
+> Participants in the TAG choose by
+> [consensus](https://www.w3.org/policies/process/#def-Consensus) their Chair or
+> co-Chairs; in the absence of consensus, the Team appoints the Chair or
+> co-Chairs of the TAG. The Chair or co-Chairs must be selected from the
+> participants of the TAG. Chair selection must be run at least at the start of
+> each regular term, as well as when a majority of the participants request it;
+> and may be run at other times when initiated by the current chairs or the
+> Team, for example if a chair steps down or if a minority of the participants
+> make such a request.
+
+While the TAG can decide to use a different chair-picking process in the future,
+so far we believe the following one is likely to work well:
+
+1. Select an outgoing TAG member to coordinate the process for the upcoming TAG.
+   If there are no outgoing members, choose someone who isn't offering to chair
+   for the year.
+1. Between the election and the start of the term, collect all members' thoughts
+   on what qualities they want in the next year's chairing, and which slates of
+   people they think would be good or bad choices. These thoughts can be
+   expressed privately to the coordinator, who will anonymize and aggregate them
+   before presenting them to the full TAG.
+1. At the earlier of the start of the term, or the point when the term's [TAG
+   appointments](https://www.w3.org/policies/process/#TAG-appointments) are
+   ratified, the coordinator publicizes all the thoughts and proposed slates
+   that they've received, and starts trying to drive the discussion toward
+   consensus on one of the slates.
+1. If no consensus is achieved by 1 month after the start of the term, ask the
+   Team to appoint.
+
+## Chairing the TAG
+
+This is not a full chairing guide, but rather a few points for chairs to keep in
+mind.
+
+* Chairs should not run discussions about topics they're heavily involved in. If
+  no other TAG chair is available, the chair(s) should delegate to another
+  volunteer to chair the discussion.
```

Comment:
```suggestion
  volunteer to chair the discussion.

* At the end of their terms, Chairs are invited to optionally share a report on their experiences and recommendations with the group.
```

This sentence could be massaged but the intention is to work towards preserving some "institutional knowledge" from the chairs' perspective.